documentation/reference/vespa-cmdline-tools.html

```diff
@@ -1499,11 +1500,141 @@ <h2 id="vespa-status-filedistribution">vespa-status-filedistribution</h2>
 
 
 
+
+
 <h2 id="vespa-visit">vespa-visit</h2>
 <p>
 Refer to <a href="../content/visiting.html">visiting</a>.
+</p><p>
+Synopsis: <code>vespa-visit [options]</code>
+</p><p>
+Options:
+<table class="table">
+  <thead></thead><tbody>
+    <tr>
+      <th style="width:300px">--abortonclusterdown</th>
+      <td>Abort if cluster is down</td>
+    </tr><tr>
+      <th>-b, --maxbuckets &lt;num&gt;</th>
+      <td>Maximum buckets per visitor</td>
+    </tr><tr>
+      <th>--bucketspace &lt;space&gt;</th>
+      <td>
+        <a href="../content/buckets.html#bucket-space">Bucket space</a> to visit
+        (<em>default</em> or <em>global</em>). If not specified, <em>default</em> is used
+      </td>
+    </tr><tr>
+      <th>-c, --cluster &lt;cluster&gt;</th>
+      <td>Visit the given cluster</td>
+    </tr><tr>
+      <th>-d, --datahandler &lt;target&gt;</th>
+      <td>Send results to the given target</td> <!-- ToDo document vespa-visit-target below and link to it -->
+    </tr><tr>
+      <th>-f, --from &lt;timestamp&gt;</th>
+      <td>Only visit from the given timestamp (microseconds)</td> <!-- ToDo: Does this work now? -->
```

Comment:
As far as I know timestamp ranges should still be supported. It's effectively just a filter in the backend and therefore doesn't affect any of the distribution/id scheme stuff.